app/controllers/postController.js

```diff
@@ -0,0 +1,19 @@
+const Posts = require('../models/posts');
+const { uploadFile } = require('../helpers/aws/uploader');
+const { S3_BUCKET_NAME: bucketName } = require('config');
+
+
+const createPost = async (data, file) => {
+    console.log('CreatePost Data Controller...');
```

Comment:
restrict to use console logger, instead, use either centralized or file-based logging.

Reply:
It will be updated in further commits